app/src/main/res/layout/contract_detail_documents_document.xml

```diff
@@ -1,10 +1,45 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Button xmlns:android="http://schemas.android.com/apk/res/android"
+<com.google.android.material.card.MaterialCardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    style="?materialButtonContainedLargeStyle"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:layout_marginStart="@dimen/base_margin_double"
-    android:layout_marginTop="@dimen/base_margin_double"
-    android:layout_marginEnd="@dimen/base_margin_double"
-    tools:text="@string/MY_DOCUMENTS_INSURANCE_CERTIFICATE" />
+    android:layout_marginBottom="@dimen/base_margin"
+    android:layout_height="80dp">
```

Comment:
Can we allow this height to naturally come from the size of the content?

Reply:
this is just temporary. Im waiting for another text to put here

Reply:
Makes sense.